### PR TITLE
Kan 110 building api

### PR DESCRIPTION
**Pull request by Binupaatu**
Registration API

### CODE REVIEW

File: backend/customers/.env.dev
Comment (Vikx001) [inline backend/customers/.env.dev:1]: Reviewed

File: backend/gateway/package.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
   "description": "This application will host the gateway point of the entire application microservices",
   "main": "index.js",
   "scripts": {
-    "start": "NODE_ENV=prod nodemon index.js",
-    "dev": "NODE_ENV=dev nodemon index.js",
+    "start": "SET NODE_ENV=prod&&nodemon index.js",
+    "dev": "SET NODE_ENV=dev&&nodemon index.js",
```
Comment (Vikx001) [inline backend/gateway/package.json:8]: Try not to add package.json file

File: backend/users/.env.dev [hunk 2]
```diff
@@ -0,0 +1,8 @@
+PORT=8881
+DB_HOST='localhost'
+DB_NAME='studioghibli'
+DB_USER='root'
+DB_PASSWORD='rootadmin'
```
Comment (Vikx001) [inline backend/users/.env.dev:5]: Try not to push pwds in dev

**Review by Vikx001 — APPROVED** [review]
Code reviewed

File: backend/users/.env.dev (hunk 2 above)
Comment (Binupaatu) [reply backend/users/.env.dev:5]: it will help other developers to use the file. I have rename the file with .dev

Comment (Vikx001) [reply backend/users/.env.dev:5]: Okay.

**Pull request merged**